jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/TaskCommon.java

```diff
@@ -63,7 +87,8 @@ static void disableHttpLogging() {
     OutputEventListener defaultOutputEventListener = context.getOutputEventListener();
     context.setOutputEventListener(
         event -> {
-          LogEvent logEvent = (LogEvent) event;
+          org.gradle.internal.logging.events.LogEvent logEvent =
```

Comment:
I don't think I know what this is about?

Reply:
Gradle and Jib both have classes named `LogEvent`, so I opted to use the full name for this one.

Reply:
oh I figured it out.